pkg/networkservice/chains/chainstest/doc.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package chainstest provides API for testing NSM chains such as Forwarder, NSC, NSMGR, NSE.
```

Comment:
@edwarnicke , @haiodo Here I suggesting to add `chainstest` pkg that provides a possibility to test real nsmgr, nsc, nse, registries via grpc unit tests. This can be super useful for testing the logic of interdomain/multi nodes chains and so on.

Note: this pkg has a name convention like `https://golang.org/pkg/net/http/httptest/`. 

Please share your thoughts about this approach :)